checkstyle.xml

```diff
@@ -0,0 +1,168 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC "-//Puppy Crawl//DTD Check Configuration 1.2//EN" "http://www.puppycrawl.com/dtds/configuration_1_2.dtd">
+<module name="Checker">
+    <property name="charset" value="UTF-8"/>
+    <property name="fileExtensions" value="java"/>
```

Comment:
You should set the language to English here since Checkstyle uses horrible machine translations that don't make any sense (see FabricMC/fabric#478)

Reply:
Ah, need to apply the same to loom as well then. (Where I took this from) 👍

Reply:
Yep, I did that in Architectury Loom (actually it was only my personal fork at that point, Arch didn't exist) but never PR'd it to upstream: https://github.com/architectury/architectury-loom/blob/cd8822457ea5c2b2f0c2a2530c4cf16d36eb965d/checkstyle.xml#L6-L7